guide.css

```diff
@@ -10,9 +10,35 @@ img{
 }
 
 .sectionTitle{
+	background-color: #FBEFD5;
+	border:none;
+	cursor:pointer;
 	text-align: center;
 	font-size: 1.5em;
 	border-bottom: 2px solid gray;
+	width:100%;
+}
+
+.sectionTitle:hover {
+	background-color: #ccc;
+}
+
+.sectionTitle:after {
+	content: "\2796"; /* Unicode character for "minus" sign (-) */
+	font-size: 13px;
+	color: white;
+	float: right;
+	margin-left: 5px;
+}
+
+/* Collapsible section content. Note: visible by default */
+.collapsible-content {
```

Comment:
All other compound titles are in camelCase, we should probably keep that theme

Reply:
Changed in 7e6afadfff67b96118a4411de22f7e26c54a3d34